.gitlab/ci/global.yml

```
@@ -68,3 +68,156 @@
   extends:
     - .default-cache
     - .default-before_script
+
+.notify-slack:
+  after_script:
+    - echo "=====Notify Slack====="
+    - |
+      if [ -n "${SLACK_ONLY_IF}" ]; then
+        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
+      else
+        echo "Skipping Slack"
+      fi
+
+run-unittests-and-lint:
+  needs: []
+  stage: unittests-and-validations
+  rules:
+    - if: '$FORCE_BUCKET_UPLOAD != "true" && $INSTANCE_TESTS != "true"'
+  artifacts:
+    expire_in: 48 hrs
+    paths:
+      - /builds/xsoar/content/unit-tests
+      - /builds/xsoar/content/artifacts/*
+  extends:
+    - .default-job-settings
+    - .notify-slack
+  variables:
+    SLACK_TEST_TYPE: "unittests"
+    SLACK_ENV_RESULTS: $ARTIFACTS_FOLDER/env_results.json
+    SLACK_ONLY_IF: $BUCKET_UPLOAD
+  script:
+    - echo "====Test Infrustracture===="
+    - python3 -m pytest ./Tests/scripts/infrastructure_tests/ -v
+    - python3 -m pytest ./Tests/Marketplace/Tests/ -v
+    - python3 -m pytest ./Tests/scripts/utils/tests -v
+    - python3 -m pytest ./Tests/tests -v
+    - python3 -m pytest ./Tests/private_build/ -v
+    - python3 -m pytest Utils -v
+    - |
+      if [ -n "${DEMISTO_SDK_NIGHTLY}" ] ; then
+      ./Tests/scripts/sdk_pylint_check.sh
+      fi
+    - echo "====Run Unit Testing and Lint===="
+    - |
+      if [[ -n $FORCE_BUCKET_UPLOAD || -n $BUCKET_UPLOAD ]] && [[ "$(echo "$GCS_MARKET_BUCKET" | tr '[:upper:]' '[:lower:]')" != "marketplace-dist" ]]; then
+        echo "Skipping validations when uploading to a test bucket."
+      else
+        echo "demisto-sdk version: $(demisto-sdk --version)"
+        echo "mypy version: $(mypy --version)"
+        echo "flake8 py2 version: $(python2 -m flake8 --version)"
+        echo "flake8 py3 version: $(python3 -m flake8 --version)"
+        echo "bandit py2 version: $(python2 -m bandit --version 2>&1)"
+        echo "bandit py3 version: $(python3 -m bandit --version 2>&1)"
+        echo "vulture py2 version: $(python2 -m vulture --version 2>&1)"
+        echo "vulture py3 version: $(python3 -m vulture --version 2>&1)"
+        SHOULD_LINT_ALL=$(./Tests/scripts/should_lint_all.sh)
+        mkdir ./unit-tests
+        if [ -n "$SHOULD_LINT_ALL" ]; then
+          echo -e  "----------\nLinting all because:\n${SHOULD_LINT_ALL}\n----------"
+          demisto-sdk lint -p 8 -a -q --test-xml ./unit-tests --log-path $ARTIFACTS_FOLDER --failure-report $ARTIFACTS_FOLDER
+        else
+          demisto-sdk lint -p 8 -g -v --test-xml ./unit-tests --log-path $ARTIFACTS_FOLDER --failure-report $ARTIFACTS_FOLDER
+        fi
+      fi
+
+run-validations:
```

Same idea as what I said for `run-unittests-and-lint`.
```suggestion
.run-validations:
```